cmd/rpcdaemon/commands/eth_call.go

```diff
@@ -411,10 +412,18 @@ func (api *APIImpl) GetWitness(ctx context.Context, blockNrOrHash rpc.BlockNumbe
 	if debug != nil {
 		dbg = *debug
 	}
-	return api.getWitness(ctx, api.db, blockNrOrHash, dbg)
+	return api.getWitness(ctx, api.db, blockNrOrHash, rpc.BlockNumberOrHashWithNumber(0), dbg)
 }
 
-func (api *BaseAPI) getWitness(ctx context.Context, db kv.RoDB, blockNrOrHash rpc.BlockNumberOrHash, debug bool) (hexutility.Bytes, error) {
+func (api *APIImpl) GetBlockRangeWitness(ctx context.Context, blockNrOrHash rpc.BlockNumberOrHash, endBlockNrOrHash rpc.BlockNumberOrHash, debug *bool) (hexutility.Bytes, error) {
```

Comment:
it would be amazing if we can move all witness-related stuff to `eth_call_zkevm.go`

Reply:
Sure I will move them `eth_call_zkevm.go`